src/app/pages/user/user.component.html

```diff
@@ -51,7 +51,7 @@
           <ng-container matColumnDef="editDetails">
             <mat-header-cell *matHeaderCellDef>עריכת פרטים</mat-header-cell>
             <mat-cell *matCellDef="let row">
-              <button mat-icon-button color="primary" (click)="updateUser(row._id, row.firstname, row.lastname, row.email, row.username)">ערוך</button>
+              <button mat-icon-button color="primary" (click)="updateUser(row._id, row.firstname, row.lastname, row.email, row.username,row.role, row.Class)">ערוך</button>
```

Comment:
There are a lot of things that are moved to the function. Maybe we can move the row object and take specific things that we need?

Reply:
I believe I'd better do it on another, separate PR.
That way the current one will stay slim and focused,
and the new one will apply the suggested tip on the other occurences, too.

What do you think?